pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -617,8 +628,23 @@ func (k *Deployer) getKptLiveInitArgs() []string {
 		flags = append(flags, "--inventory-id", k.Live.Apply.InventoryID)
 	}
 
+	flags = append(flags, k.getGlobalFlags()...)
+	return flags
+}
+func (k *Deployer) getGlobalFlags() []string {
+	var flags []string
+
+	if k.kubeContext != "" {
+		flags = append(flags, "--context", k.kubeContext)
+	}
+	if k.kubeConfig != "" {
+		flags = append(flags, "--kubeconfig", k.kubeConfig)
+	}
 	if len(k.Live.Apply.InventoryNamespace) > 0 {
```

Comment:
should we document that `InventoryNamespace` overrides the value provided by the cmd flag `--namespace`?

Reply:
I'm thinking of removing this inventoryNamespace actually. Maybe in the following PRs.